README.md

```diff
@@ -91,6 +91,7 @@ Contributors are Welcome. Please read the [Contribution](https://github.com/mbj3
 
 - [Airhorn](https://github.com/GoogleChrome/airhorn) - Airhorn
 - [Weather pwa-app](https://github.com/googlecodelabs/your-first-pwapp) - Weather-app
+- [Service workers - The Chromium Projects](https://www.chromium.org/blink/serviceworker) - Chromium Project
```

Comment:
I don't find this a open source project. It's a link to understand about service workers. U should add this in 'Learning Section'